package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "cross-env NODE_ENV=test && mocha src/test --require @babel/register --recursive --exit || true",
+    "start": "node ./src/index.js",
```

Comment:
Hi @oyewoas, This might error because we are using es6 which is compiled by babel into a build directory. I think you need to run build script while starting the container,  and then point the start script to the build directory

Reply:
we are not running docker on the local machine with that line, this script will be needed when docker is to be hosted remotely on docker hub.

To run docker on local machine we just use  *docker-compose up*

Reply:
Alright @oyewoas, We just have to keep that in mind anyway